utils/workspace/uncertainties.py

```diff
@@ -212,6 +212,35 @@ def get_pu_shape(year: str, analysis: str) -> dict[str, Any]:
     return systematics[analysis][year]
 
 
+def get_qcd_estimate_shape(year: str, analysis: str) -> dict[str, Any]:
+    """Return prefiring shape systematics for a given year and analysis."""
+    systematics = {
+        "vbf": {"Run3": {}},
+        "monojet": {
+            "Run3": {
+                syst: {"signal": {"value": 1.0, "processes": get_processes_by_region(analysis=analysis, region="signal", types=["data_driven"])}}
+                for syst in [f"qcdfit_{analysis}_{year}", f"qcdbinning_{analysis}_{year}"]
+            },
+        },
+    }
+    return systematics[analysis][year]
+
+
+def get_diboson_shape(year: str, analysis: str) -> dict[str, Any]:
+    """Return prefiring shape systematics for a given year and analysis."""
+    systematics = {
+        "vbf": {"Run3": {}},
+        "monojet": {
+            "Run3": {
+                f"{vv}_ewkqcd_mix": {region: {"value": 1.0, "processes": [vv]} for region in ["signal", "dimuon", "dielec", "singlemu", "singleel"]}
```

Comment:
what if we use `get_all_regions`? I think photon will be ignored if there is no that sample in that region.